controllers/ApiController.php

```diff
@@ -813,7 +813,7 @@ protected function checkSingleObjectWriteVersion($objectType, $obj=null, $json=f
 	 */
 	protected function checkLibraryIfUnmodifiedSinceVersion($required=false) {
 		if (Z_CONFIG::$TESTING_SITE && !empty($_GET['skipetag'])) {
```

Comment:
Just to make manual testing of successive writes with curl a little easier, without updating the version each time. I don't think I've used it in years, but for now can just change the `continue` to `return true`.